### ISSUE TITLE
[Bug Report] Wrong cursor on [aria-disabled] selector

### ISSUE THREAD

**Issue by lmcsu** [vuetifyjs/vuetify#11003]
### Environment
**Vuetify Version:** 2.2.20
**Vue Version:** 2.6.11
**Browsers:** Chrome 80.0.3987.149
**OS:** Windows 10

### Steps to reproduce
Add [aria-disabled="false"] on any element

### Expected Behavior
Cursor shouldn't change

### Actual Behavior
Cursor changes to "default"

### Reproduction Link
<a href="https://codepen.io/letmecode/pen/gOpybMG" target="_blank">https://codepen.io/letmecode/pen/gOpybMG</a>

### Other comments
The selector in _reset.scss for "cursor: default" should be `[aria-disabled="true"]` instead of `[aria-disabled]`. Currently it also applies to `[aria-disabled="false"]`

Also, according to the specification https://www.w3.org/WAI/PF/aria/states_and_properties#aria-disabled the default value for "aria-disabled" is "false"

**Comment by clandau** [comment]
I'm going to work on this as a first issue, working on getting the process down. Hope that's ok.
